style.css

```diff
@@ -363,3 +357,92 @@ label {
   line-height: 32px;
   color: #42526e;
 }
+
+/* desktop version //////////////////// */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 1440px;
+  }
+
+  /* headline /////////////////////////// */
+
+  .app-bar-and-headline-container {
+    background-image: url(images/backgrounds/desktop-header-llustration.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+  }
+
+  .toolbar {
+    display: none;
+  }
+
+  .toolbar-desktop {
+    display: flex;
+    justify-content: center;
+    column-gap: 50px;
+  }
+
+  .headline {
+    width: 920px;
+    background-image: none;
+    margin: 0 auto;
+  }
+
+  /* recent-works  /////////////////////// */
+
+  .recent-works .text {
+    padding: 100px 0 40px 0;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .card-work-row {
+    grid-template-columns: repeat(3, 1fr); /* 3 columns with equal width */
+    grid-template-rows: repeat(2, 1fr);
+    column-gap: 24px;
+    padding: 0 134px;
+  }
+
+  /* about me /////////////////////////// */
+
+  .about-me {
+    padding: 60px 134px;
+  }
+
+  .about-text {
+    padding: 0 122px;
+  }
+
+  .supporting-text-1 {
+    padding: 0 70px;
+  }
+
+  .about-cards {
+    grid-template-columns: repeat(3, 1fr); /* 3 columns with equal width */
+    column-gap: 24px;
+  }
+
+  .about-me .about-text {
+    padding-bottom: 50px;
+  }
+
+  /* CONTACT-FORM /////////////////////// */
+
+  .contact-form {
+    flex-direction: row;
+    background-image: url(images/backgrounds/desktop-contact.svg);
+  }
+
+  .contact-form-text {
+    padding: 0 40px;
+  }
```

Comment:
- [ ] Kindly correct the position of the text in the contact form, in the Figma template it is upper than in your design. 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/220759317-b1df8ef3-e850-4cfe-b903-18919b5ae33b.png) | ![image](https://user-images.githubusercontent.com/58642949/220759200-1dd24705-1c7f-4229-80cb-ea8f80b16be3.png) |